scripts/parse.py

```diff
@@ -18,13 +18,17 @@
 # along with taserver.  If not, see <http://www.gnu.org/licenses/>.
 #
 
+from typing import Optional, Set
 import io
 import os
 import struct
 import sys
 import time
 
+from taconstants import TAConstants, read_taconstants
+
 last_seen_seqnr = None
+knownTAConstants = read_taconstants('known_constants.txt')
```

Comment:
Don't put code in global scope if you can avoid it. I think in this case we should put all the parse methods in a parse class and make knownTAConstants a member variable to avoid having to pass it around everywhere. infile and outfile could also become member variables.

Reply:
Yeah, agreed. I put it there because otherwise it would need to be passed in everywhere.

Refactoring into a class and making it a member variable should solve that.